index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portofolio</title>
+  </head>
+  <body>
+    <!--The main container: all elements are in-->
+    <main id="id-main-content">
+      <!--the document header-->
+      <header id="id-header-content">
+        <img src="/images/ic_menu.svg" alt="menu button" id="id-side-menu">
+      </header>
+
+      <!--this section contain information about the author-->
+      <section id="id-main-section">
+        <p id="id-my-name">Marc Raphael</p>
```

Comment:
This is like the title of your portfolio. You should use the h1 tag here. Kindly make the requested change.